### PR TITLE
Added ButaneDAO

### PR DESCRIPTION
**Pull request by bbcchain**
#### Hi! What is your PR about?

- [ ] Add or edit a skin
- [x] Add a custom domain for your space
- [x] Add an alias to migrate your space

### CODE REVIEW

File: spaces/aliases.json [hunk 1]
```diff
@@ -62,6 +62,7 @@
   "tptdao.eth": "itokenpocket.eth",
   "tallyhodao.eth": "tahodao.eth",
   "goodghosting.eth": "halofi.eth",
+  "butanedao.eth": "butanedao.eth",
```
Comment (gokaiorg) [inline spaces/aliases.json:65]: Use aliases.json when you aim to proceed to a space migration.
```suggestion
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: spaces/aliases.json (hunk 1 above)
Comment (bbcchain) [reply spaces/aliases.json]: > Use aliases.json when you aim to proceed to a space migration.

I have updated the PR

File: spaces/aliases.json [hunk 2]
```diff
@@ -62,6 +62,7 @@
   "tptdao.eth": "itokenpocket.eth",
   "tallyhodao.eth": "tahodao.eth",
   "goodghosting.eth": "halofi.eth",
+  "vote.butanedao.io": "butanedao.eth",
```
Comment (gokaiorg) [inline spaces/aliases.json:65]: You don't need to edit the aliases file, it is reserved for space migrations. 
```suggestion
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Review by bbcchain — COMMENTED** [review]
I hope this is correct

**Comment by gokaiorg** [comment]
> I hope this is correct

Nope 😁 By editing the aliases file you mean you want to migrate https://snapshot.org/#/butanedao.io to https://snapshot.org/#/butanedao.eth. But the butenedao.io space does not exist.

**Comment by bbcchain** [comment]
I deleted the space thinking I could redo it. Im trying to get vote.butanedao.io to work. We have also added our chain in chain list. Could you please make necessary changes?

Thank you

**Comment by gokaiorg** [comment]
I can't find your new space. Please make sure that you have filled in all required fields and added a strategy. https://snapshot.org/#/butanedao.eth

**Comment by gokaiorg** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.

**Pull request closed without merging**